src/e2e/app/cli/repo-tests/src/test_flow.rs

```diff
@@ -736,6 +746,903 @@ pub async fn test_dataset_trigger_flow(kamu_api_server_client: KamuApiServerClie
         .await;
 }
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_trigger_flow_ingest(kamu_api_server_client: KamuApiServerClient) {
+    let temp_dir = tempfile::tempdir().unwrap();
```

Comment:
I think these tests could benefit from another layer of abstraction over GQL API calls.
Currently it's really hard to follow what is happening in the test in general, because it's quite long, and uses GQL low-level templating. 
The GQL calls are not very specific to particular tests, these are just possible "steps" in the API, and the templates with parameterization could live in another utility layer.

Reply:
Even if these GQL templates would only be used in any other test suites, and they don't qualify to become a more common test API, there will still be a benefit from a module-local helper. Currently there is signifficant duplication in the way flows are triggered and their results are matched against the expected outcome.

Reply:
Fair point -- I'll make it easier to read

Reply:
A little rethinking of some of the methods -- I think it's gotten better: what do you think?